src/main.rs

```diff
@@ -54,6 +54,11 @@ async fn main() -> stable_eyre::Result<()> {
         )
         .try_init()?;
 
+    // Use ring as default crypto provider
+    rustls::crypto::ring::default_provider()
+        .install_default()
+        .expect("Failed to install rustls crypto provider");
```

Comment:
```suggestion
        .wrap_err("Failed to install rustls crypto provider")?;
```

So we have the full error stack in eyre